plugin/manager.go

```diff
@@ -191,6 +215,24 @@ func (m *manager) Start() error {
 	return nil
 }
 
+func (m *manager) startInterceptors() error {
+	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		handlerName := r.Header.Get("Vcluster-Plugin-Handler-Name")
```

Comment:
Can we rename the handler name to `VCluster-Plugin-Handler-Name`

Reply:
Sure, will have to update the header in vcluster afterwards though